src/components/shift-schedule/AdministratorMenu.tsx

```diff
@@ -0,0 +1,58 @@
+'use client';
+
+import { Button } from '@/components/ui/Button';
+import {
+  Collapsible,
+  CollapsibleContent,
+  CollapsibleTrigger,
+} from '@/components/ui/Collapsible';
+import { ChevronDownIcon, ChevronUpIcon, Trash2Icon } from 'lucide-react';
+import { useState } from 'react';
+
+type AdministratorMenuProps = {
+  messages: {
```

Comment:
For consistency, when passing down translations as prop call the object for `t` instead of `messages` since that is what we have done everywhere else